calling-apis/problem-set-requests-with-js.md

```diff
@@ -0,0 +1,296 @@
+# Problem Set: Requests With JS
+
+## Directions
+
+Read the problem prompt, which will ask you to write JavaScript code to find certain answers.
+
+Once you found the answers, input them into the answers prompt below.
+
+## Preparation: LocationIQ Forward Geocoding API
+
+Use the [LocationIQ's Forward Geocoding API](https://locationiq.com/geocoding). Before starting this problem prompt:
+
+1. Obtain an API key for LocationIQ's Forward Geocoding API
+1. Familiarize yourself with the LocationIQ API Documentation
+1. Familiarize yourself with the `axios` library
+1. Open Postman for manual testing
+
+![Screenshot of LocationIQ's feature on the Geocoding API](../assets/calling-apis_problem-set-requests-with-js_location-iq-geocoding.png)
+
+## Problem Prompt
+
+Create a JavaScript program that finds the latitude and longitude coordinates for the [seven wonders of the New7Wonders of the World](https://en.wikipedia.org/wiki/New7Wonders_of_the_World), which are the following:
+
+| Wonder              |
+| ------------------- |
+| Great Wall of China |
+| Petra               |
+| Colosseum           |
+| Chichen Itza        |
+| Machu Picchu        |
+| Taj Mahal           |
+| Christ the Redeemer |
+
+### Extra Challenges
+
+See if you can accomplish these extra challenges. These challenges are tricky and interesting! One solution required extra research around the following subjects: `setTimeout`, "JavaScript sleep," `async` and `await`, and (traditional) `for` loops.
```

Comment:
Just say that researching these topics may be required rather than referencing some solution they can't see.
```suggestion
See if you can accomplish these extra challenges. These challenges are tricky and interesting! Implementing these extensions may require extra research around the following subjects: `setTimeout`, "JavaScript sleep," `async` and `await`, and (traditional) `for` loops.
```